angular/src/directives/angular-component-lib/utils.ts

```diff
@@ -4,46 +4,40 @@ import { fromEvent } from 'rxjs';
 
 export const proxyInputs = (Cmp: any, inputs: string[]) => {
   const Prototype = Cmp.prototype;
-  inputs.forEach(item => {
+  inputs.forEach((item) => {
```

Comment:
Double checking....should these format changes be in this PR? I didn't see a callout about it in the summary || any prettier changes in `package.json`

Reply:
Good catch -- I added this to the original PR. I updated the output target deps, and it looks like there were some formatting changes: https://github.com/ionic-team/stencil-ds-output-targets/blob/f20a7780bf6416052e338fd31bd1a813477f9792/packages/angular-output-target/angular-component-lib/utils.ts#L7